src/Authentication/Authentication.Test/Helpers/AuthenticationHelpersTests.cs

```diff
@@ -32,7 +32,7 @@ public AuthenticationHelpersTests()
         public async Task ShouldUseDelegateAuthProviderWhenUserAccessTokenIsProvidedAsync()
         {
             // Arrange
-            string accessToken = "ACCESS_TOKEN_VIA_DELEGATE_PROVIDER";
+            string accessToken = "eyJhbGciOiJIUzI1NiJ9.eyJSb2xlIjoiVGVzdCIsIklzc3VlciI6Iklzc3VlciIsIlVzZXJuYW1lIjoiVGVzdCIsImV4cCI6MTY3ODQ4ODgxNiwiaWF0IjoxNjc4NDg4ODE2fQ.hpYypwHAV8H3jb4KuTiLpgLWy9A8H2d9HG7SxJ8Kpn0";
```

Comment:
@peombwa Why do we have a hard coded token?

Reply:
@timayabi2020, this is a dummy access token to running the test. Here is the jwt.ms output:
``` string
{
  "alg": "HS256"
}.{
  "Role": "Test",
  "Issuer": "Issuer",
  "Username": "Test",
  "exp": 1678488816,
  "iat": 1678488816
}.[Signature]
```

As part of this PR, `Connect-MgGraph` will now validate the structure of the JWT to determine if it can be decoded as per https://tools.ietf.org/html/rfc7519.